docs/frontend/cookbook/new_module.md

```diff
@@ -1 +1,141 @@
-# Create New Module
\ No newline at end of file
+# Create New Module
+
+#### Adding a new module often involves adding some new business space or adding a mechanism to modify existing modules.
+
+---
+
+The process of adding new modules can take place in the development version and in the implementation version:
+- The **development** process is related to the contribution to the [frontend repository][git-dev].
+- The **implementation** process is related to the installation of [skeleton-frontend repository][git-skelet].
+
+Adding a new module in these two approaches is very similar and the differences will be described.<br>
+
+> The example will be built based on the **implementation process**. <br>
+For example, in the module we will create a new **basic page** with **routing** and a side **menu entry**.
+
+1. In the project we create a directory of `modules`, of course if it no longer exists.
```

Comment:
1. In the project create a `modules` directory. - Consider as it would not exist